File: gin-vue-admin/web/src/api/ffEmployee.js
import service from '@/utils/request'

export const getEmployeeList = (params) => {
  return service({
    url: '/ffProduct/getEmployeeList',
    method: 'get',
    params
  })
}

export const createEmployee = (data) => {
  return service({
    url: '/ffProduct/createEmployee',
    method: 'post',
    data
  })
}

export const updateEmployee = (data) => {
  return service({
    url: '/ffProduct/updateEmployee',
    method: 'put',
    data
  })
}

export const deleteEmployee = (data) => {
  return service({
    url: '/ffProduct/deleteEmployee',
    method: 'delete',
    data
  })
}

File: gin-vue-admin/server/router/ffproduction/ff_employee.go
package ffproduction

import (
	"gin-vue-admin/server/middleware"
	"github.com/gin-gonic/gin"
)

type EmployeeRouter struct{}

func (e *EmployeeRouter) InitEmployeeRouter(Router *gin.RouterGroup) {
	employeeRouter := Router.Group("ffProduct").Use(middleware.OperationRecord())
	employeeRouterWithoutRecord := Router.Group("ffProduct")
	{
		employeeRouter.POST("createEmployee", employeeApi.CreateEmployee)
		employeeRouter.PUT("updateEmployee", employeeApi.UpdateEmployee)
		employeeRouter.DELETE("deleteEmployee", employeeApi.DeleteEmployee)
	}
	{
		employeeRouterWithoutRecord.GET("getEmployeeList", employeeApi.GetEmployeeList)
	}
}
